packages/osc-ecommerce/app/components/InstantSearch/Components/Cards/InstantSearchCourseCard.tsx

```diff
@@ -56,13 +57,28 @@ export const InstantSearchCourseCard = (props: CourseCardProps) => {
     const PriceTag = (
         <CardPriceTag>
             <>
-                {/* TODO: REPLACE THIS WITH ETIKA DATA */}
-                <Price size="sm">
-                    <span className="u-text-bold">DATA FROM ETIKA</span>/month
-                </Price>
-                <Price size="sm">
-                    or from <span className="u-text-bold">{`£${hit.price} in full`}</span>
-                </Price>
+                {hit.lowestMonthlyRepayment ? (
+                    <>
+                        <Price size="sm">
+                            <span className="u-text-bold">
+                                £{Number(hit.lowestMonthlyRepayment).toFixed(2)}
```

Comment:
I think this should be prefixed with "From"

```suggestion
                                From £{Number(hit.lowestMonthlyRepayment).toFixed(2)}
```

Reply:
Added